### PR TITLE
Project files service proposal

### PR DESCRIPTION
**Pull request by ykulbak**
First cut of a project files service proposal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -984,7 +985,60 @@ Each topic has two identifiers:
...
+### 4.3.1 GET Project Files Service
+
+**Resource URL**
+
+    GET /bcf/{version}/projects/{project_id}/files
+    
+[project_files_GET.json](Schemas_draft-03/Collaboration/File/project_files_GET.json)
+
+Retrieve a **collection** of project files that supports allowing users to choose which files (models) to reference in
+the header of topics ppostyed to that the server.
```
Comment (pasi-paasiala) [inline README.md:997]: Typo in ppostyed

Comment (ykulbak) [reply README.md]: Fixed, and documentation improved. Can you please check?

File: README.md [hunk 2]
```diff
@@ -984,7 +985,60 @@ Each topic has two identifiers:
 
 ## 4.3 File Services
 
-### 4.3.1 GET Files (Header) Service
+### 4.3.1 GET Project Files Service
+
+**Resource URL**
+
+    GET /bcf/{version}/projects/{project_id}/files
```
Comment (pasi-paasiala) [inline README.md:992]: Maybe the endpoint could be named to getFileInformation.

Comment (ykulbak) [reply README.md]: @pasi-paasiala @GeorgDangl fixed in the latest commit, I think this pull request is ready to merge.

File: Schemas_draft-03/Collaboration/File/project_file_display_information.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    "field_display_name": {
+      "required": true,
+      "type": "string"
+    },
+    "field_value": {
+      "required": true,
+      "type": "string"
+    }
+  }
+}
```
Comment (jasollien) [inline Schemas_draft-03/Collaboration/File/project_file_display_information.json:16]: Very minor, but add a newline here :)

**Comment by jasollien** [comment]
One minor comment. Rest looks good :)

**Comment by pasi-paasiala** [comment]
I think the change looks good, but I'd squash the commits to one, so it is easier to understand and we don't have unnecessary commits in the release branch.

**Review by pasi-paasiala — APPROVED** [review]
Just squash the commits.

File: Schemas_draft-03/Collaboration/File/project_file_display_information.json (hunk 3 above)
Comment (ykulbak) [reply Schemas_draft-03/Collaboration/File/project_file_display_information.json]: done

**Pull request merged**